website/content/en/docs/deployment/rds-s3/guide.md

```diff
@@ -160,16 +188,129 @@ yq e '.rds.secretName = env(RDS_SECRET)' -i charts/common/aws-secrets-manager/rd
             {{< /tab >}}
             {{< /tabpane >}} 
 
+### 2.4 Manual S3 Setup
+   1. Export values:
+      ```bash
+      export S3_SECRET="<your s3 secret name>"
+      export S3_BUCKET="<your s3 bucket name>"
+      export MINIO_SERVICE_HOST=s3.amazonaws.com
+      ```
+
+   1. [IRSA ONLY] IRSA Roles
+
+      1. Associate an OpenID Connect (OIDC) provider with your IAM role to authenticate your cluster with the IAM service.
+
+         ```bash
+         eksctl utils associate-iam-oidc-provider --cluster ${CLUSTER_NAME} \
+         --region ${CLUSTER_REGION} --approve
+         ```  
+      2. Get the following OIDC information for future reference:
+         ```bash
+         export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+         export OIDC_PROVIDER_URL=$(aws eks describe-cluster --name $CLUSTER_NAME --region $CLUSTER_REGION \
+         --query "cluster.identity.oidc.issuer" --output text | cut -c9-)
+         ```
+
+      3. [Create an IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) to scope the permissions for the Role. For simplicity, we will restrict access to the specified S3_BUCKET as an example.
```

Comment:
Create an IAM Policy with access to the S3 bucket where pipeline artifacts will be stored. The following policy grants full access to the S3 bucket, you can scope it down by giving read, write and GetBucketLocation permissions.